test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -69,6 +69,10 @@ private static class ClassConfiguration {
 		 */
 		private ClassConfiguration valueClassUsedInCode;
 
+		private String[] preloadClasses;
+		private Integer implicitCreationFlags;
+		private boolean hasNullRestrictedAttribute;
```

Comment:
Is there a reason to use a `boolean` for this ?

Reply:
I chose boolean because the NullRestricted attribute does not carry any additional data.

Reply:
Isn't this attribute associated with a field rather than a class ?